website/server/libs/cron.js

```diff
@@ -165,6 +165,11 @@ function awardLoginIncentives (user) {
       count += 1;
     }
 
+    // Overwrite notificationData.rewardText if rewardName was explicitly declared
+    if (loginIncentive.rewardName) {
+      notificationData.rewardText = i18n.t(loginIncentive.rewardName);
```

Comment:
can you pass user.preferences.language as the second parameter here? Otherwise it'll always be in English. And since you're there can you also add it to the i18n.t call at line 160?